readme.md

```diff
@@ -112,23 +112,23 @@ instanceC.toPlural(); // money (no change)
 
 * **Verb conjugation**
 	1. Dictionary lookup (known irregularities + 4000 common verbs)
-	2. If the passed verb is identified as infinitive then applies regular expression transformations that are based on word endings, vowels and consonant phonetics.
+	2. If the passed verb is identified as infinitive it then applies regular expression transformations that are based on word endings, vowels and consonant phonetics.

```

Comment:
Should also insert a comma after "infinitive," beause the sentence starts with a dependent (subordinate) clause.